packages/rum-recorder/src/boot/recorder.spec.ts

```diff
@@ -159,59 +161,20 @@ describe('startRecording', () => {
       })
     })
   })
-})
-
-describe('trackFocusRecords', () => {
-  let hasFocus: boolean
-  let addRecordSpy: jasmine.Spy<(rawRecord: RawRecord) => void>
-  let lifeCycle: LifeCycle
-
-  beforeEach(() => {
-    hasFocus = true
-    lifeCycle = new LifeCycle()
-    spyOn(Document.prototype, 'hasFocus').and.callFake(() => hasFocus)
-    addRecordSpy = jasmine.createSpy()
-  })
-
-  it('adds an initial Focus record', () => {
-    trackFocusRecords(lifeCycle, addRecordSpy)
-    expect(addRecordSpy).toHaveBeenCalled()
-  })
-
-  it('adds a Focus record on focus', () => {
-    trackFocusRecords(lifeCycle, addRecordSpy)
-    addRecordSpy.calls.reset()
-
-    window.dispatchEvent(createNewEvent('focus'))
-    expect(addRecordSpy).toHaveBeenCalled()
-  })
-
-  it('adds a Focus record on blur', () => {
-    trackFocusRecords(lifeCycle, addRecordSpy)
-    addRecordSpy.calls.reset()
-
-    window.dispatchEvent(createNewEvent('blur'))
-    expect(addRecordSpy).toHaveBeenCalled()
-  })
-
-  it('adds a Focus record on new view', () => {
-    trackFocusRecords(lifeCycle, addRecordSpy)
-    addRecordSpy.calls.reset()
-
-    lifeCycle.notify(LifeCycleEventType.VIEW_CREATED, {} as any)
-    expect(addRecordSpy).toHaveBeenCalled()
-  })
 
-  it('set has_focus to true if the document has the focus', () => {
-    hasFocus = true
-    trackFocusRecords(lifeCycle, addRecordSpy)
-    expect((addRecordSpy.calls.mostRecent().args[0] as FocusRecord).data.has_focus).toBe(true)
-  })
+  // eslint-disable-next-line max-len
+  it('does not split Meta, Focus and FullSnapshot records between multiple segments when taking a full snapshot', (done) => {
+    setMaxSegmentSize(10)
```

Comment:
idk what is the size for meta, focus and full snapshot so not sure if 10 means something specific.
could it be 0?
naming the 10 value could help understand what we are trying to do here

Reply:
Alright